scripts/GenSongsList.py

```diff
@@ -93,30 +77,14 @@ def findMatchingBasename(files, basename):
 #       else:
 #         htmlOutput.write(f"  <a href=\"{str(i)}\">{ext(i)}</a>\n")
 #
-        if ext(i).lower() == ".pdf":
-          htmlOutput.write(f"  <a href=\"{str(i)}\">{ext(i)}</a>\n")
-        elif ext(i).lower() == ".chopro":
-          htmlOutput.write(f"  <a href=\"{str(i)}\" download>{ext(i)}</a>\n")
-        elif ext(i).lower() == ".cho":
+        if ext(i) == ".url":
+          with open(i, "r") as urlFile:
+            address = urlFile.readline()
+          htmlOutput.write(f"<a href=\"{address}\">video</a>\n")
+        elif (ext(i).lower() == ".chopro") or (ext(i) == ".cho") or (ext(i) == ".mscz"):
```

Comment:
Since there seems to be only two kinds of html getting printed (either the 'download' version or not), I just collapsed this into a shorter elif chain, where all the "download" extensions get "or"ed together